package/src/components/Message/MessageSimple/utils/renderText.tsx

```diff
@@ -141,8 +141,15 @@ export const renderText = <
       : Linking.canOpenURL(url).then((canOpenUrl) => canOpenUrl && Linking.openURL(url));
   };
 
+  let previousLink: string;
   const linkReact: ReactNodeOutput = (node, output, { ...state }) => {
-    const url = node.target;
+    let url: string;
+    if (state?.withinLink) {
```

Comment:
```suggestion
    if (state?.withinLink && previousLink) {
```

also add comments for some explanation here please

Reply:
Done